src/include/daos_srv/container.h

```diff
@@ -120,6 +120,7 @@ struct ds_cont_child {
 	uint32_t		 sc_status_pm_ver;
 	/* flag of CONT_CAPA_READ_DATA/_WRITE_DATA disabled */
 	uint32_t		 sc_rw_disabled:1;
+	daos_epoch_t		 sc_migrate_epoch;
```

Comment:
It's better to add some comments to explain the purpose of this 'sc_migrate_epoch'.

Reply:
OK, I will add some comment.